tests/models/covid19/awardSpendingAgency/BaseAwardSpendingByAgencyRow-test.js

```diff
@@ -0,0 +1,37 @@
+import BaseAwardSpendingByAgencyRow from "models/covid19/awardSpendingAgency/BaseAwardSpendingByAgencyRow";
+
+export const mockBaseAwardSpendingByAgencyRow = {
+    id: 41,
+    description: "Description text of 012, for humans",
+    children: [],
+    count: 2,
```

Comment:
Recommend making this a value >= 1000 to test that it gets formatted with commas